docs/tutorials/mac_experimental.md

```diff
@@ -0,0 +1,86 @@
+# Using podman-machine on MacOS (x86_64 and Apple silicon)
+
+## SetUp
+
+You must obtain a compressed tarball that contains the following:
+* a qcow image
+* a podman binary
+* a gvproxy binary
+
+You must also have installed brew prior to following this process.  See https://brew.sh/ for
+installation instructions.
+
+Note: If your user has admin rights, you can ignore the use of `sudo` in these instructions.
+
+
+1. Install qemu from brew to obtain the required runtime dependencies.
+
+   ```
+   brew install qemu
+   ```
+
+2. If you are running MacOS on the Intel architecture, you can skip to step 8.
+3. Uninstall the brew package
+
+   ```
+   brew uninstall qemu
+   ```
+
+4. Get upstream qemu source code.
+
+   ```
+   git clone https://github.com/qemu/qemu
+   ```
+
+5. Apply patches that have not been merged into upstream qemu.
+
+   ```
+   cd qemu
+   git checkout v5.2.0
+   curl https://patchwork.kernel.org/series/418581/mbox/ | git am --exclude=MAINTAINERS
+   curl -L https://gist.github.com/citruz/9896cd6fb63288ac95f81716756cb9aa/raw/2d613e9a003b28dfe688f33055706d3873025a40/xcode-12-4.patch | git apply -
+   ```
+
+6. Install qemu build dependencies
+
+   ```
+   brew install libffi gettext pkg-config autoconf automake pixman ninja make
+   ```
+
+7. Configure, compile, and install qemu
+   ```
+   mkdir build
+   cd build
+   ../configure --target-list=aarch64-softmmu --disable-gnutls
+   gmake -j8
+   sudo gmake install
+   ```
+
+
+8. Uncompress and place provided binaries into filesystem
+
+   **Note**: In the following instructions, you need to know the name of the compressed file
+that you were given.  It will be used in two of the steps below.
+
+   ```
+   cd ~
+   tar xvf `compressed_file_ending_in_xz`
+   sudo cp -v `unpacked_directory`/{gvproxy,podman} /usr/local/bin
+   ```
+
+9. Sign all binaries
+
+   ```
+   sudo codesign --entitlements ~/qemu/accel/hvf/entitlements.plist --force -s - /usr/local/bin/qemu-* /usr/local/bin/gvproxy /usr/local/bin/podman
+   ```
+
+
+## Test podman
+
+1. run podman machine init --image-path /path/to/image
```

Comment:
can we drop the "run" here and just turn these into commands to copy-paste?

Reply:
Yes

```
1.
podman machine init --image-path /path/to/image
```